src/View/Widgets/AppMenu.vala

```diff
@@ -112,6 +112,17 @@ public class Files.AppMenu : Gtk.Popover {
         };
         show_remote_thumbnails.get_style_context ().add_class (Gtk.STYLE_CLASS_MENUITEM);
 
+        ///TRANSLATORS The format of the date (possibly with time) shown in the Modified column of the file view
+        var datetimeformat_header = new Granite.HeaderLabel (_("Date & Time Format"));
+
+        var iso_button = new Gtk.RadioButton.with_label (null, DateFormatMode.ISO.to_string ());
+        iso_button.get_style_context ().add_class (Gtk.STYLE_CLASS_MENUITEM);
+        var locale_button = new Gtk.RadioButton.with_label_from_widget (iso_button, DateFormatMode.LOCALE.to_string ());
+        locale_button.get_style_context ().add_class (Gtk.STYLE_CLASS_MENUITEM);
+        var informal_button = new Gtk.RadioButton.with_label_from_widget (iso_button, DateFormatMode.INFORMAL.to_string ());
+        informal_button.get_style_context ().add_class (Gtk.STYLE_CLASS_MENUITEM);
+        //TODO Add a custom format wizard? Or a detailed informat format? Or "days ago" format?
```

Comment:
I think we should simply remove this TODO and file an issue if we won't address it in this PR but really want to in the future. TODO comments are often buried in projects that have many source codes and are developed by multiple developers.

Reply:
OK, I've now limited this PR to exposing the setting.  We can consider other formats later.